qa/L0_logging/test.sh

```diff
@@ -60,6 +60,11 @@ source ../common/util.sh
 rm -f *.log
 rm -fr $MODELSDIR && mkdir -p $MODELSDIR
 
+if [ ! -d ${DATADIR} ]; then
```

Comment:
Is this check added just to let us know if the model repository in the dlcluster is not setup correctly? or if there's other cases that we want to catch?

Reply:
I ran the test locally with a non-existing datadir and the test ran and failed much later with obscure errors -- so just to catch low hanging fruit later on for future debuggers